env/HERA.env

```diff
@@ -72,12 +72,12 @@ elif [ $step = "anal" ]; then
     export NTHREADS_CYCLE=${nth_cycle:-12}
     [[ $NTHREADS_CYCLE -gt $npe_node_max ]] && export NTHREADS_CYCLE=$npe_node_max
     npe_cycle=${ntiles:-6}
-    export APRUN_CYCLE="$launcher -n $npe_cycle"
+    export APRUN_CYCLE="$launcher -N 1 -n $npe_cycle"
```

Comment:
Is hardwiring `-N 1` the way to go?  What happens if one wishes to spread it across multiple nodes?

Reply:
The current design is only use N 1 configuration. Multiple change will need to be done if across multiple nodes is become a requirement. The code manager can make that change as need. The N 1 solution is representing the fact of this job in the current state.